fleet/fleet.go

```diff
@@ -217,3 +256,6 @@ func (f fleet) ipFromUnitState(unitState *schema.UnitState) (net.IP, error) {
 
 	return nil, maskAny(ipNotFoundError)
 }
+
+// TODO
```

Comment:
> What do I miss?

Separation of concern. This is only used by the cli and has no use elsewhere. Why does the cli even need to know about this fleet package?

> Would implementing String() on UnitStatus work?

Yes, but then you again have cli logic (I want the status to be build like this) in the fleet package.